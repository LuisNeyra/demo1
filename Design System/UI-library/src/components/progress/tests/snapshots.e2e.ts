import { snapIt } from '../../../test';

const component = 'progress';

describe(component, () => {
  const snap = snapIt(component);

  describe('renders', () => {
    snap('basic progress bar', '<blaze-progress percentage="5">5%</blaze-progress>');
    snap('size correctly', '<blaze-progress size="small" percentage="15">15%</blaze-progress>');
    snap('type correctly', '<blaze-progress size="medium" type="info" percentage="20">20%</blaze-progress>');
    snap(
      'rounded ends correctly',
      '<blaze-progress size="large" type="success" percentage="25" rounded>25%</blaze-progress>'
    );
  });
});
