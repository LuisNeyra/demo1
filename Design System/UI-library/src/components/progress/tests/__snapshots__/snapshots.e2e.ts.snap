// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`progress renders basic progress bar 1`] = `"<blaze-progress percentage=\\"5\\" class=\\"hydrated\\"><!----><div class=\\"c-progress\\">5%</div></blaze-progress>"`;

exports[`progress renders rounded ends correctly 1`] = `"<blaze-progress size=\\"large\\" type=\\"success\\" percentage=\\"25\\" rounded=\\"\\" class=\\"hydrated\\"><!----><div class=\\"c-progress c-progress--rounded u-large\\">25%</div></blaze-progress>"`;

exports[`progress renders size correctly 1`] = `"<blaze-progress size=\\"small\\" percentage=\\"15\\" class=\\"hydrated\\"><!----><div class=\\"c-progress u-small\\">15%</div></blaze-progress>"`;

exports[`progress renders type correctly 1`] = `"<blaze-progress size=\\"medium\\" type=\\"info\\" percentage=\\"20\\" class=\\"hydrated\\"><!----><div class=\\"c-progress u-medium\\">20%</div></blaze-progress>"`;
